### ISSUE TITLE
Deactivate button does not reflect deactivation in Metamask

### ISSUE THREAD

**Issue by FBosler** [EthWorks/useDApp#412]
Cool repo!

# Bugreport:
The deactivate button does not seem to propagate to Metamask (I have not tested for other wallets). 

## After connecting:
![image](https://user-images.githubusercontent.com/14016698/142760692-bd2b4cd8-858a-4e53-a083-be3f11da6f9f.png)

## After disconnecting:
![image](https://user-images.githubusercontent.com/14016698/142760704-0345cc76-0b7f-4e64-bdb0-ee4988fcc32a.png)

Find a minimal working example [here](https://github.com/FBosler/usedapp-deactivate-bug).

**Comment by hirokimoto** [comment]
Yeah I noticed this error, too. I have deactivated my metamask account, but this package still returns the address of my metamask. It means that it actually has not deacticated the connection.

**Comment by gsteenkamp89** [comment]
I might be wrong, but I believe calling `deactivate()` simply clears the web3-react state machine (from `useEthers()` in this package).
So `{ account, connector }` etc. you destructure from `useEthers()` will be cleared.
The fact that Metamask still shows "connect" just means you have previously granted permission for Metamask to connect to this domain and inject the web3 provider into the browser to be used for reading/writing to some network. You can always revoke this permission from your wallet.

**Comment by hirokimoto** [comment]
@FBosler I strongly feel that you have 2 more accounts in your metamask. I think you have deactivated the app from the account 1, but I think that you have not deacticated the app from account 2.

**Comment by appleseed-iii** [comment]
yes I have the same issue

**Comment by ccarnino** [comment]
Same issue here.

**Comment by mellydecyber** [comment]
yeahh, I'm facing the same issue

**Comment by LongTran12** [comment]
same issue!

**Comment by mj426382** [comment]
Metamask does not share `deactivating from page` api.
